distributed/tests/test_dask_collections.py

```diff
@@ -109,6 +110,13 @@ async def test_dask_array_collections(c, s, a, b):
         np.testing.assert_equal(o_local, o_remote)
 
 
+@pytest.mark.skipif(
+    (
+        parse_version(dask.__version__) < parse_version("2023.2.2")
+        and parse_version(dask.__version__) >= parse_version("2023.2.1")
+    ),
+    reason="https://github.com/dask/dask/pull/10005",
+)
 @gen_cluster(client=True)
 async def test_bag_groupby_tasks_default(c, s, a, b):
```

Comment:
I decided to skip this test to unblock our CI. https://github.com/dask/dask/pull/10005 will introduce the same test in dask/dask, i.e. we could even remove it from distributed. keeping it here for early failures is still nice since the test is very cheap